content/guides/set-up-rails-appplication-with-omniauth-bookingsync.md

```diff
@@ -0,0 +1,303 @@
+# Building New Rails Application With Devise And BookingSync Integration
+
+If you need to integrate your existing Rails application (which most likely uses Devise for authentication) with BookingSync, that's the exact guide you are looking for.
+
+## Setup
+
+After setting up environment for Ruby (check either [rvm](https://rvm.io) or [rbenv](https://github.com/rbenv/rbenv)), install `bundler` and `rails` gems:
+
+~~~
+gem install bundler
+gem install rails
+~~~
+
+And generate new Rails application:
+
+~~~
+rails new omniauth-bookingsync-app
+cd omniauth-bookingsync-app
+~~~
+
+To finish the setup we just need to create the database:
+
+~~~
+bundle exec rake db:create
+~~~
+
+To verify it works, just run:
+
+~~~
+rails server
+~~~
+
+And visit `http://localhost:3000`.
+
+
+## Adding Devise
+
+Add [`devise`](https://github.com/plataformatec/devise) gem to Gemfile:
+
+~~~ Gemfile
+gem 'devise'
+~~~
+
+And run:
+
+~~~
+bundle install
+~~~
+
+Then run the generator:
+
+~~~
+rails generate devise:install
+rails generate devise:views
+~~~
+
+We don't need to have a signup / signin alternative with BookingSync, just a way to integrate the account, so we can remove this part from `app/views/devise/shared/_links.html.erb`:
+
+~~~ html app/views/devise/shared/_links.html.erb
+<%- if devise_mapping.omniauthable? %>
+  <%- resource_class.omniauth_providers.each do |provider| %>
+    <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider) %><br />
+  <%- end -%>
+<% end -%>
+~~~
```

Comment:
Not really a comment, it's a part of the docs, just not sure why it doesn't really work for html / erb.